go/cmd/pfacct/radius.go

```diff
@@ -160,7 +160,8 @@ func (h *PfAcct) handleAccountingRequest(rr radiusRequest) {
 			ok = false
 		}
 		if !ok {
-			logError(ctx, "handleAccountingRequest: accouting queue - max capacity reached")
+			logError(ctx, "handleAccountingRequest: accounting queue - max capacity reached")
+			return
```

Comment:
Will that return but **not answer** so that the controller retries?

Otherwise we miss out on the information in that packet

Reply:
At least i think i need to requeue it since the radius reply has already been sent.